doc/modules/mod_pubsub.md

```diff
@@ -118,6 +118,45 @@ This node type **requires** `publish-options` with at least `device_id` and `ser
 
 If you'd like to learn more about metrics in MongooseIM, please visit the [MongooseIM metrics](../operation-and-maintenance/Mongoose-metrics.md) page.
 
+#### Overall PubSub action metrics
+
+For every PubSub action, like node creation, subscription, publication there are following kind of metrics:
```

Comment:
For every PubSub action, like node creation, subscription and publication, the following metrics are available: